bom/pom.xml

```diff
@@ -44,7 +44,7 @@
     <jmeter.version>3.3</jmeter.version>
     <junit.version>4.12</junit.version>
     <logback.version>1.2.3</logback.version>
-    <micrometer.version>1.0.9</micrometer.version>
+    <micrometer.version>1.1.3</micrometer.version>
```

Comment:
there's already version 1.1.4 available ...

Reply:
Looks like this came out after I filed the CQs, I will update the PR.